h/static/scripts/annotator/plugin/enhancedanchoring.coffee

```diff
@@ -47,7 +50,7 @@ class Anchor
       # Don't call TextHighlight directly; instead, make a system
       # For registering highlight creators, or publish an event, or
       # whatever
-      @highlight[page] = @Annotator.TextHighlight.createFrom @_getSegment(page), this, page
+      @highlight[page] = Anchor.Annotator.TextHighlight.createFrom @_getSegment(page), this, page
```

Comment:
> I think this got lost in all the changes,
>

Nah, it was waiting in the queue.

> but just a note to remove this and cache a module local reference to
> |Annotator| at the top of the document.
>

Thx, did that in
https://github.com/hypothesis/h/commit/85ea94a6117c892e6a5fbdca1de064fc86d71632